plugins/callback/scalyr.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Martin Migasiewicz <migasiew.nk@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+callback: scalyr
+type: aggregate
+short_description: Sends task result events to Scalyr
+author: Martin Migasiewicz (@martinm82)
+version_added: 1.3.0
+description:
+  - This callback plugin will send task results as JSON formatted events to Scalyr.
+requirements:
+  - Whitelisting this callback plugin
```

Comment:
```suggestion
  - Whitelisting this callback plugin.
```
Sentences in `description` should always be complete, i.e. end with some kind of punctuation.